crates/codegen/src/get_node_properties.rs

```diff
@@ -559,6 +559,28 @@ fn custom_handlers(node: &Node) -> TokenStream {
                 tokens.push(TokenProperty::from(Token::Authorization));
             }
         },
+        "DefineStmt" => quote! {
+            tokens.push(TokenProperty::from(Token::Create));
+            match n.kind() {
+                ObjectType::ObjectAggregate => tokens.push(TokenProperty::from(Token::Aggregate)),
+                _ => todo!("Unknown DefineStmt {:#?}", n.kind())
+            }
+            if let Some(node) = &n.args[0].node {
+                if let NodeEnum::List(n) = node {
+                    if n.items.len() == 2 {
+                        tokens.push(TokenProperty::from(Token::Order));
+                        tokens.push(TokenProperty::from(Token::By));
+                        tokens.push(TokenProperty::from(Token::Ident));
+                        tokens.push(TokenProperty::from(Token::Ident));
```

Comment:
As discussed: Not sure why `Token::Ident` should be pushed twice (or else parser panics)

Reply:
To reproduce: comment the 3 `tokens.push(TokenProperty::from(Token::Ident))` in `DefineStmt` and run `cargo test valid_statements`

Parser panics twice for the same token with the same span:

```sh
could not find node for token Token { kind: Ident, text: "anyelement", span: 485..495, token_type: NoKeyword } at depth 2
```